style-guide.md

```diff
@@ -0,0 +1,41 @@
+# Technical documentation writing guidelines
+
+Technical documentation avoids conversational language because it must be translatable, accessible, and understood by a variety of audiences. Effective technical writing is direct, bland, and bossy! (And that's ok!)
+
+In addition to “normal” grammar rules, IBM Quantum documentation follows [IBM style guidelines](https://ibmdocs-test.dcs.ibm.com/docs/en/ibm-style), which include rules for ensuring clear translation, correct [terminology usage](https://w3.terminology.g11n.ibm.com/standards/terminology/), accessibility, and general guidelines that helps all IBM documentation have the same look and feel. (Note that this is specific to technical documentation, not marketing material or blogs, which have their own voice, tone, and style guides.)
+
+The following list highlights grammatical and stylistic situations we most often encounter while proofreading documentation.
+
+## Titles and headers
+
+- Titles and headers use "sentence case" (first letter of the first word is capitalized; the rest lower-case) as opposed to "camel case" (where each word is capitalized)
```

Comment:
This is the convention we use in programming to make more clear what each means:

```suggestion
- Titles and headers use "Sentence case" (first letter of the first word is capitalized; the rest lower-case) as opposed to "Camel Case" (where each word is capitalized)
```

Reply:
I appreciate the effort to be visually illustrative in this case but I can't bring myself to capitalize things randomly in the middle of a sentence :D

Reply:
I brought myself to it 🤣